### ISSUE TITLE
Pytyhon2/Python3.6 legacy support doesn't work

### ISSUE THREAD

**Issue by puremourning** [puremourning/vimspector#665]
### Description

https://github.com/puremourning/vimspector/issues/573#issuecomment-1241245285

The path in the adapter spec is wrong. Vimspector installs the right legacy debugpy but then runs the new one.

### Works in vim?

Reproducable in Vim

### Works with clean config?

No

### Sample project works?

Reproducable with sample project

### Minimal Reproduction

:VimspectorInstall debugpy-python2
use a python 3.6 pyenv and run up support/test/python/simple_python/main.py
F5
Select the -python2 adapter

### Expected Behaviour

legacy adapter is started, and debugging works

### Actual Behaviour

fails

### Installation Type

Plugin Manager (git clone)

### Vimspector version

HEAD

### Vim version

```Text
9.0.350
```

### Declaration

- [X] I have read and understood [CONTRIBUTING.md](https://github.com/puremourning/vimspector/blob/master/CONTRIBUTING.md#diagnostics)
- [X] I have read and understood the [CODE_OF_CONDUCT.md](https://github.com/puremourning/vimspector/blob/master/CODE_OF_CONDUCT.md)

**Comment by puremourning** [comment]
Actual repro steps involve a small change to the launch config, which explains why I didn't find this when I tested it:

```
ben@BeniMac2020 simple_python % git diff .
diff --git a/support/test/python/simple_python/.vimspector.json b/support/test/python/simple_python/.vimspector.json
index 1e6e953..2fa2e89 100644
--- a/support/test/python/simple_python/.vimspector.json
+++ b/support/test/python/simple_python/.vimspector.json
@@ -129,6 +129,7 @@
       "adapter": "debugpy-python2",
       "configuration": {
         "request": "launch",
+        "python": "${pathToPython:python}",
         "type": "python",
         "cwd": "${workspaceRoot}",
         "program": "${file}",
```

As we use sys.executable when installing the gadgets

**Comment by axrt** [comment]
Thank you for looking into this, I actually agree that the legacy python may not need to be supported, and can offer to create a pr to add another item [here](https://github.com/puremourning/vimspector/wiki/Additional-Language-Support) listing the debugpy for adapter and configuration.

**Comment by puremourning** [comment]
For now I just fixed it in #666

**Comment by axrt** [comment]
Thank you!
